src/fields/components/InternalField.tsx

```diff
@@ -13,6 +15,16 @@ interface InternalFieldProps extends InternalCommonFieldProps {
   fieldType: 'field' | 'fieldset'
 }
 
+export const statusContents: {
+  [key in Status]: {
+    icon: string
+    color: Color
+    textContent: string
+  }
+} = {
+  complete: { icon: 'included', color: 'success', textContent: 'Complete' },
```

Comment:
We discussed possibly adding another status so I took the approach of adding a flexible status prop. We could even potentially deprecate error in the future and include in this. 

We can also simplify all this and just make the prop display complete status by default.

Reply:
What other status other than complete we might wanna have?
Deprecating error might take quite a lot of time considering how widely uses is and the need of a dynamic error message might not fit with the way status has been implemented so I'm wondering if it's actually worth it going down this route?

Reply:
So the designs considered adding incomplete status but we decided not to include it for now.